Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -54,7 +54,7 @@ class Digitizer
 
   std::vector<HitType> mHitContainer; // The container of hits in a given detector
 
-  bool getHitContainer(const int, const std::vector<HitType>&, std::vector<HitType>&); // True if there are hits in the detector
+  void getHitContainerPerDetector(const std::vector<HitType>&, std::array<std::vector<HitType>, 540>&);
```

Comment:
could we make a constant with a good name for 540? this will make the code more readable.

Reply:
Thanks for pointing that out, I had it in mind and it just went out like this. It is fixed now. Instead of including the full CommonParams.h header file here I just added on extra enum. Is that ok?